Ajout du poste RFIUM après l'AG du 5 février

laura-salas opened the pull request.

laura-salas committed:
Ajout du poste RFIUM après l'AG du 5 février

tienne-B commented on charte.tex:
```diff
@@ -160,6 +160,7 @@ \subsection{Composition du Conseil d'Administration}\label{sec:composition-du-co
 \item Coordonnateur à la vie étudiante
 \item Représentant du café étudiant Math-Info
 \item Représentant des étudiant·e·s internationaux
+\item Représentant.e Regroupement de Femmes en Informatique (RFIUM)
```
```suggestion
\item Représentant·e du Regroupement de Femmes en Informatique
```
Point au milieu (constance)

tienne-B commented on charte.tex:
```diff
@@ -181,6 +182,44 @@ \subsubsection{Représentant du café étudiant Math-Info}\label{sec:representan
 La tâche du représentant du café étudiant est définie dans la charte du café. Il doit former un comité constitué de membres de l'AÉDIROUM pour remplir les postes requis par la charte du café. Il représente l'administration du café devant l'assocation et le Conseil d'Administration devant l'administration du café.
 
+
+\subsubsection{Représentant.e Regroupement de Femmes en Info. (RFIUM)}\label{sec:representant.e-regroupement-de-femmes-en-info-rfium}
```
```suggestion
\subsubsection{Représentant·e du Regroupement de Femmes en Informatique}\label{sec:representant-e-regroupement-de-femmes-en-informatique}
```

tienne-B commented on charte.tex:
```diff
@@ -181,6 +182,44 @@ \subsubsection{Représentant du café étudiant Math-Info}\label{sec:representan
+
+\vspace{0.3cm}
+
+\textbf{Définitions:}\\
+\begin{enumerate}
+    \item \textbf{Femme:} Toute personne s'identifiant comme femme. 
+    \item \textbf{RFIUM:} Regroupement de Femmes en Informatique de l'Université de Montréal
+\end{enumerate}
```
Une section "globale" des définitions serait bien, pas juste dans un sous-sous-section. (Dans la section "Généralités")

Ce serait une bonne utilisation de
```latex
\begin{description}
\item[Femme] ...
```

tienne-B commented on charte.tex:
```diff
@@ -181,6 +182,44 @@ \subsubsection{Représentant du café étudiant Math-Info}\label{sec:representan
+
+\vspace{0.3cm}
+
+\textbf{Description générale du rôle:}
```
Je ne metterais pas d'en-tête ici

tienne-B commented on charte.tex:
```diff
@@ -181,6 +182,44 @@ \subsubsection{Représentant du café étudiant Math-Info}\label{sec:representan
+
+\vspace{0.3cm}
+
+S'assure de représenter les intérêts et préoccupations du Regroupement de Femmes en Informatique de l'Université de Montréal. Notamment, lors de toute assemblée générale, conseil administratif ou tout moment de prise de décision auquel participerait un ou plusieurs membres du conseil administratif de l'AÉDIROUM, la responsable aura comme responsabilité de:\\
```
```suggestion
S'assure de représenter les intérêts et préoccupations du Regroupement de Femmes en Informatique de l'Université de Montréal. Notamment, lors de toute assemblée générale, conseil administratif ou tout moment de prise de décision auquel participerait un ou plusieurs Membres du Conseil d'Administration, le·a responsable aura comme responsabilité de~:

```

tienne-B commented on charte.tex:
```diff
@@ -181,6 +182,44 @@ \subsubsection{Représentant du café étudiant Math-Info}\label{sec:representan
+    \item Être l'image du RFIUM au sein de l'AÉDIROUM, et vice-versa 
+\end{itemize}
+
+\vspace{0.3cm}
+
+\textbf{Conditions du rôle:}
+
+\vspace{0.3cm}
```
```suggestion
\paragraph{Conditions d'éligibilité}
```

`paragraph` vient après `subsubsection`

tienne-B commented on charte.tex:
```diff
@@ -181,6 +182,44 @@ \subsubsection{Représentant du café étudiant Math-Info}\label{sec:representan
+
+\vspace{0.3cm}
+
+Doit être recommandée par le RFIUM avant de postuler à L'AÉDIROUM, avec approbation de 50\%+1 vote, lors d'une AG ou CA du RFIUM ayant eu lieu préalablement, soit durant la même session ou durant la session précédant celle-ci.  Le RFIUM doit avoir élu au moins 5 membres de son conseil administratif au moment de l'élection.  
```
Pour être pédantique, "AG" ni "CA" sont définis... Faudrait peut-être les mettre dans la nouvelle section "Définitions" :P

tienne-B reviewed (changes requested):
Quelques commentaires et suggestions de codification. Faut essayer d'éviter d'utiliser des commandes stylistiques, en préférence s'il y a des commandes sémantiques.

laura-salas committed:
Update charte.tex

Co-authored-by: Étienne Beaulé <beauleetienne0@gmail.com>

laura-salas committed:
Update charte.tex

Co-authored-by: Étienne Beaulé <beauleetienne0@gmail.com>

laura-salas committed:
Section definition + ajustements structurels

- Ajouté une section "définitions" au début de la charte pour clarifier certains termes et abréviations 
- Raffinements structurels

tienne-B commented on charte.tex:
```diff
@@ -10,6 +10,15 @@
 
+\subsection{Définitions}
+\begin{description}
+    \item \textbf{AG:} Assemblée générale
+    \item \textbf{CA:} Conseil administratif
+    \item \textbf{Femme:} Toute personne s'identifiant comme femme. 
+    \item \textbf{RFIUM:} Regroupement de Femmes en Informatique de l'Université de Montréal
```
```suggestion
    \item \textbf{AG} Assemblée générale;
    \item \textbf{CA} Réunion du Conseil d'Administration;
    \item \textbf{Femme} Toute personne s'identifiant comme femme;
    \item \textbf{RFIUM} Regroupement de Femmes en Informatique de l'Université de Montréal.
```

tienne-B reviewed (approved):
Petit peu de style encore, mais c'est bon.

laura-salas committed:
Rajouter acronyme RFIUM

Pour faciliter la reconnaissance du groupe sous son acronyme

laura-salas committed:
Update charte.tex

Co-authored-by: Étienne Beaulé <beauleetienne0@gmail.com>
